src/main/telemetry/frsky.c

```diff
@@ -121,6 +121,7 @@ extern int16_t telemTemperature1; // FIXME dependency on mw.c
 
 static uint32_t lastCycleTime = 0;
 static uint8_t cycleNum = 0;
+escAndServoConfig_t *escAndServoConfig;
```

Comment:
You should use masterConfig.escAndServoConfig. 

BTW: escAndServoConfig should be declared static in src/main/flight/altitudehold.c ..